url/resources/urltestdata.json

```diff
@@ -3303,12 +3303,14 @@
   {
     "input": "http:@:www.example.com",
     "base": "about:blank",
-    "failure": true
+    "failure": true,
+    "inputCanBeRelative": true
```

Comment:
Testing with https://jsdom.github.io/whatwg-url/#url=aHR0cDpAOnd3dy5leGFtcGxlLmNvbQ==&base=aHR0cHM6Ly9leGFtcGxlLmNvbS8= implies this is correct as-is

Reply:
ditto